README.md

```diff
@@ -5,3 +5,15 @@
 [![Discussion Forum](https://img.shields.io/discourse/https/forum.vaticle.com/topics.svg)](https://forum.vaticle.com)
 [![Stack Overflow](https://img.shields.io/badge/stackoverflow-typedb-796de3.svg)](https://stackoverflow.com/questions/tagged/typedb)
 [![Stack Overflow](https://img.shields.io/badge/stackoverflow-typeql-3dce8c.svg)](https://stackoverflow.com/questions/tagged/typeql)
+
+This repository stores all TypeDB Drivers built by Vaticle.
+
+See the table below for links to README files, documentation, and source codes.
+
+| Driver  | Readme                                                                             | Documentation                                                                 | Source codes                                                                                         |
+|---------|------------------------------------------------------------------------------------|-------------------------------------------------------------------------------|------------------------------------------------------------------------------------------------------|
+| Rust    | [README](https://github.com/vaticle/typedb-driver/tree/development/rust/README.md) |                                                                 | `rust/src` [directory](https://github.com/vaticle/typedb-driver/tree/development/rust/src)           |
```

Comment:
Rust documentation will be out very soon.

Reply:
We can either 
- merge it now without the link and add the link later, when it's ready, or 
- add a link now, expecting it to be eventually working, but showing 404 for now. 
I'd prefer the first variant. What do you think?

Reply:
Definitely the former, please.

Reply:
I added a link to rust docs that do not exist yet - https://github.com/vaticle/typedb-driver/pull/453/commits/c3a6e092b602e3e83294e3287478356b384106f9